modules/renter/upload.go

```diff
@@ -68,6 +69,26 @@ func validateSiapath(siapath string) error {
 	return nil
 }
 
+// validateSource verifies that a sourcePath meets the
+// requirements for upload.
+func validateSource(sourcePath string) error {
+	f, err := os.Open(sourcePath)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	finfo, err := f.Stat()
```

Comment:
Oops, thought this was in the API package. The name is fine for the renter package